scripts/zones/Windurst_Waters_[S]/npcs/Door_Rhinostery.lua

```diff
@@ -0,0 +1,50 @@
+-----------------------------------
+--  Area: Windurst Waters (S)
+--   NPC: Door
+-- @zone: 94
+
+-----------------------------------
+package.loaded["scripts/zones/Windurst_Waters_[S]/TextIDs"] = nil;
+-----------------------------------
+
+-----------------------------------
+-- onTrade Action
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end;
+
+-----------------------------------
+-- onTrigger Action
+-----------------------------------
+
+function onTrigger(player,npc)
+	if (player:getQuestStatus(CRYSTAL_WAR, THE_LOST_BOOK)==QUEST_AVAILABLE and player:hasKeyItem(925)) then
+	player:startEvent(143);
+	player:addQuest(CRYSTAL_WAR, THE_LOST_BOOK); 
+	elseif player:hasKeyItem(971) then
+	player:startEvent(144);
+	elseif (player:getQuestStatus(CRYSTAL_WAR, THE_LOST_BOOK)==QUEST_ACCEPTED) then
+	end
```

Comment:
4 space indent for each item within control block (applies to control blocks within control blocks too)